rspace/src/main/scala/coop/rchain/rspace/RSpace.scala

```diff
@@ -197,22 +196,16 @@ class RSpace[F[_]: Concurrent: ContextShift: Log: Metrics: Span, C, P, A, K](
       continuation: K,
       persist: Boolean,
       peeks: SortedSet[Int]
-  ): F[Consume] = syncF.delay {
-    eventLog.update(consumeRef +: _)
-    consumeRef
-  }
+  ): F[Consume] = eventLog.update(consumeRef +: _).as(consumeRef)
 
   protected override def logProduce(
       produceRef: Produce,
       channel: C,
       data: A,
       persist: Boolean
-  ): F[Produce] = syncF.delay {
-    eventLog.update(produceRef +: _)
-    if (!persist)
-      produceCounter.update(_.putAndIncrementCounter(produceRef))
-    produceRef
-  }
+  ): F[Produce] = eventLog.update(produceRef +: _).map(_ =>
+        if (!persist) produceCounter.update(_.putAndIncrementCounter(produceRef))).as(produceRef)
```

Comment:
We a are using linter for the code which is run when code is compile as you can see on CI output.
<https://github.com/rchain/rchain/pull/3736/checks>

Ensure that you run `sbt Test/compile` (this includes compiling the tests also) before committing the code which will ensure that `scalafmt` is run and auto formatting is done.

Reply:
I set scalafmt right now